### ISSUE TITLE
Menu Editor in UI will not load menus when UTF-8 is enabled

### ISSUE THREAD

**Issue by jdigory** [interchange/interchange#84]
Tested in IC 5.8, with Perl 5.18.1, by k2b3:
http://www.icdevgroup.org/pipermail/interchange-users/2013-November/054345.html

When clicking on a menu from the "Menu" tab in the UI, the menu will not load. An error is given:

```
192.168.1.105 KLbn8XW9:192 - [05/November/2013:10:40:40 -0500] mysite
k2b3.mydomain
.local/admin/menu_editor.html Safe: 'require' trapped by operation mask
at /usr/loc
al/perl-5.18.1/lib/5.18.1/x86_64-linux/Encode.pm line 5.
> Compilation failed in require at
/usr/local/perl-5.18.1/lib/5.18.1/x86_64-linux/E
ncode/Alias.pm line 22.
```

If the following lines in catalog.cfg are commented, menus will load just fine:

```
#Variable MV_UTF8 1
#Variable MV_HTTP_CHARSET utf-8
```

**Comment by jdigory** [comment]
the [perl] block in lib/UI/pags/admin/menu_editor.html probably needs to be moved to a UserTag to work around this

**Comment by machack666** [comment]
This should be fixed in HEAD.

**Comment by jdigory** [comment]
Fixed by b971e1ed
